.circleci/config.yml

```diff
@@ -387,11 +387,6 @@ jobs:
               echo "Skipping instance tests for forked PRs"
               exit 0
             fi
-            if [ -n "${INSTANCE_TESTS}" ] || [ $CIRCLE_NODE_INDEX -ne 0 ] ;
```

Comment:
maybe we should keep the INSTANCE_TESTS section?

Reply:
Right, Fixed